packages/auth/src/connection/Connection.ts

```diff
@@ -694,10 +694,15 @@ export class Connection extends EventEmitter<ConnectionEvents> {
     this.#machine = createActor(machine)
 
     // emit and log all transitions
-    this.#machine.subscribe(state => {
-      const summary = stateSummary(state.value as string)
-      this.emit('change', summary)
-      this.#log(`⏩ ${summary} `)
+    this.#machine.subscribe({
+      next: state => {
+        const summary = stateSummary(state.value as string)
+        this.emit('change', summary)
+        this.#log(`⏩ ${summary} `)
+      },
+      error: error => {
+        console.error('Connection encountered an error', error)
```

Comment:
I'm only logging the error to the console here. I'm not sure if it's helpful to also/instead call `this.#log`.

I also experimented with calling `this.stop()` here after logging the error to try to force disconnecting this `Connection`. I don't know if that's appropriate or if there's some better means of disconnecting. Below is a listing of the output around the exception with and without calling `stop()` for comparison:

Without calling `this.stop()`:

```
  localfirst:auth:connection:localhost:brent emit change 'connected' +1ms
  localfirst:auth:connection:localhost:brent ⏩ connected  +0ms
  localfirst:auth:connection:localhost:brent emit updated undefined +0ms
  localfirst:auth:connection:localhost admitted device +4ms
Connection encountered an error Error: Device k1ut2gm7y1kqf4was69jm6w3 not found
    at assert (file:///C:/dev/xdev/local-first-auth/packages/shared/dist/index.js:4:19)
    at device (file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:699:3)
    at memberByDeviceId (file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:891:22)
    at Team.memberByDeviceId (file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:1812:12)
    at admit (file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:2474:39)
    at file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:2478:24
    at Function.resolveAssign [as resolve] (C:\dev\xdev\local-first-auth\node_modules\.pnpm\xstate@5.9.1\node_modules\xstate\dist\log-a312ebfc.cjs.js:77:21)
    at resolveAndExecuteActionsWithContext (C:\dev\xdev\local-first-auth\node_modules\.pnpm\xstate@5.9.1\node_modules\xstate\dist\raise-182bb5c9.cjs.js:2240:56)
    at resolveActionsAndContext (C:\dev\xdev\local-first-auth\node_modules\.pnpm\xstate@5.9.1\node_modules\xstate\dist\raise-182bb5c9.cjs.js:2262:21)
    at microstep (C:\dev\xdev\local-first-auth\node_modules\.pnpm\xstate@5.9.1\node_modules\xstate\dist\raise-182bb5c9.cjs.js:1987:15)
  localfirst:auth:connection:localhost:brent <-brent #23 SYNC {"head":["3qnkp5bwi8bhv6AH8AMuP77PR2EGvQC8e6ZXB45StY7H"]} +27ms
  localfirst:auth:connection:localhost:brent sending sync message {"head":["3qnkp5bwi8bhv6AH8AMuP77PR2EGvQC8e6ZXB45StY7H"]} +1ms
  localfirst:auth:connection:localhost:brent ->brent #12 SYNC {"head":["3qnkp5bwi8bhv6AH8AMuP77PR2EGvQC8e6ZXB45StY7H"]} +0ms
  localfirst:auth:connection:localhost:brent emit change 'connected' +1ms
  localfirst:auth:connection:localhost:brent ⏩ connected  +0ms
  localfirst:auth:connection:localhost <-? #2 DISCONNECT  +7s
```


With calling `this.stop()`:

```
  localfirst:auth:connection:localhost:brent emit change 'connected' +0ms
  localfirst:auth:connection:localhost:brent ⏩ connected  +0ms
  localfirst:auth:connection:localhost:brent emit updated undefined +1ms
  localfirst:auth:connection:localhost admitted device +6ms
Connection encountered an error Error: Device hfppeln7nthulp6nlb1u1jbo not found
    at assert (file:///C:/dev/xdev/local-first-auth/packages/shared/dist/index.js:4:19)
    at device (file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:699:3)
    at memberByDeviceId (file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:891:22)
    at Team.memberByDeviceId (file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:1812:12)
    at admit (file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:2474:39)
    at file:///C:/dev/xdev/local-first-auth/packages/auth/dist/index.js:2478:24
    at Function.resolveAssign [as resolve] (C:\dev\xdev\local-first-auth\node_modules\.pnpm\xstate@5.9.1\node_modules\xstate\dist\log-a312ebfc.cjs.js:77:21)
    at resolveAndExecuteActionsWithContext (C:\dev\xdev\local-first-auth\node_modules\.pnpm\xstate@5.9.1\node_modules\xstate\dist\raise-182bb5c9.cjs.js:2240:56)
    at resolveActionsAndContext (C:\dev\xdev\local-first-auth\node_modules\.pnpm\xstate@5.9.1\node_modules\xstate\dist\raise-182bb5c9.cjs.js:2262:21)
    at microstep (C:\dev\xdev\local-first-auth\node_modules\.pnpm\xstate@5.9.1\node_modules\xstate\dist\raise-182bb5c9.cjs.js:1987:15)
  localfirst:auth:connection:localhost ->? #2 DISCONNECT  +2ms
  localfirst:auth:connection:localhost connection stopped +1ms
  localfirst:auth:connection:localhost:brent <-brent #23 SYNC {"head":["E33dZ6zvXdsA7bfpT3Ebu7jK7wZMBTTsEA7qrnoxtiVD"]} +24ms
  localfirst:auth:connection:localhost:brent sending sync message {"head":["E33dZ6zvXdsA7bfpT3Ebu7jK7wZMBTTsEA7qrnoxtiVD"]} +1ms
  localfirst:auth:connection:localhost:brent ->brent #12 SYNC {"head":["E33dZ6zvXdsA7bfpT3Ebu7jK7wZMBTTsEA7qrnoxtiVD"]} +0ms
  localfirst:auth:connection:localhost:brent emit change 'connected' +1ms
  localfirst:auth:connection:localhost:brent ⏩ connected  +0ms
```